recipes/mplview/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "mplview" %}
+{% set version = "0.0.1" %}
+{% set sha256 = "06a6cbe922f2a0d95938dcb2937b0b8fbdabd0c81b2f2ee58ee70fc616776e3b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - matplotlib
+    - npctypes
+
+test:
+  imports:
+    - mplview
+
+about:
+  home: https://github.com/jakirkham/mplview
+  license: BSD 3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Python utility functions for slices.
```

Comment:
Is this the correct summary? I think it might have been cut and pasted from another recent recipe.